receiver/hostmetricsreceiver/hostmetrics_receiver_test.go

```diff
@@ -98,12 +98,12 @@ func TestGatherMetrics_EndToEnd(t *testing.T) {
 	require.NoError(t, err, "Failed to start metrics receiver: %v", err)
 	defer func() { assert.NoError(t, receiver.Shutdown(context.Background())) }()
 
-	time.Sleep(180 * time.Millisecond)
+	time.Sleep(500 * time.Millisecond)
```

Comment:
This is going to depend on machine speed, etc.

If you need to wait for a specific condition `testutils.WaitFor` may be a better choice.

Reply:
Yea sorry this was a dirty fix while waiting for https://github.com/open-telemetry/opentelemetry-collector/pull/952

I've changed this PR to use `require.Eventually` as well